plugins/woocommerce-admin/client/customize-store/design-with-ai/state-machine.tsx

```diff
@@ -127,11 +136,23 @@ export const designWithAiStateMachineDefinition = createMachine(
 						invoke: {
 							src: 'getLookAndTone',
 							onError: {
-								actions: [ 'logAIAPIRequestError' ],
+								actions: [
+									{
+										type: 'recordTracksStepCompleted',
+										step: 'business_info_description',
+									},
+									'logAIAPIRequestError',
```

Comment:
I think we need to add a tracks event to this too, the other two tracks which I previously added only track issues with the content of the response - this one will track http call failure

Reply:
Ok, added a track in 2b5736e10061e3b1ce25b1d95675f30ad9405612